processing/src/main/java/io/druid/segment/column/SimpleDictionaryEncodedColumn.java

```diff
@@ -89,5 +89,12 @@ public int getCardinality()
   public void close() throws IOException
   {
     cachedLookups.close();
+
+    if(column != null) {
```

Comment:
You'll probably want to do the other closes even if the prior closes throw IOException